tests/components/hue_ble/test_config_flow.py

```diff
@@ -0,0 +1,627 @@
+"""Test the Hue BLE config flow."""
+
+from unittest.mock import AsyncMock, PropertyMock, patch
+
+from homeassistant import config_entries
+from homeassistant.components.hue_ble.config_flow import (
+    CannotConnect,
+    InvalidAuth,
+    NotFound,
+    ScannerNotAvailable,
+)
+from homeassistant.components.hue_ble.const import DOMAIN
+from homeassistant.const import CONF_MAC, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+from homeassistant.helpers import device_registry as dr
+
+from . import HUE_BLE_SERVICE_INFO, TEST_DEVICE_MAC, TEST_DEVICE_NAME
+
+from tests.components.bluetooth import generate_ble_device
+
+
+async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+    """Test form with good data."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
```

Comment:
```suggestion
        DOMAIN, context={"source": SOURCE_USER}
```

And import like this instead:

`from homeassistant.config_entries import SOURCE_USER`